### DISCUSSION TITLE
Export | not possible anymore if 0 entries are found

### DISCUSSION THREAD

**Discussion by relume** [kimai/kimai#3264] [General]
Version: 1.17.1

Hello

We are using the Export functionality (to PDF) for reporting on monthly basis the working-hours for our users. Also in the case if an user had not worked, thus has 0 entries, we generated since ever a PDF report/export. This seems actually not possible any more, as only the waring is shown, but the preview section with the export Buttons (csv, ..., pdf) are not shown anymore. Is there anyway to reenable this?

Many thanks in advance, 

André

**Comment by relume** [comment]
Hello

It is also the case in Version : 1.19.5.

best regards

**Reply by kevinpapst** [reply]
Yes, exporting nothing doesn't make a lot of sense...

**Comment by relume** [comment]
Hello

Thanks for your response. I agree to you that exporting no data to cvs, ods, excel makes no sense, but as report to pdf as "proof" I think it make sense.

best

**Reply by kevinpapst** [reply]
A PDF as monthly proof for controlling issues is for me something else then an empty export.

But okay, I am not entirely sure here. On the one hand I want to simplify/unify the workflows as much as possible.
On the other hand who am I to judge whether an empty report is useful for you.

Well, I will think about it. No promises for a specific result though...
